docs/sdk/src/guides/enable_elastic_scaling.rs

```diff
@@ -0,0 +1,238 @@
+//!      parachain slot duration.
+//!     - Replace the `aura::run` with the `slot_based::run` call and remove the explicit task
+//!       spawn:
+#![doc = docify::embed!("../../cumulus/polkadot-omni-node/lib/src/nodes/aura.rs", launch_slot_based_collator)]
+//!
+//! 3. In `start_parachain_node()`, destructure `slot_based_handle` from `params.other`. Remove the
+//!    `overseer_handle` and `relay_chain_slot_duration` params passed to `start_consensus` and pass
+//!    in the `slot_based_handle`.
+//!
+//! ### Phase 2 - Configure core selection policy in the parachain runtime
+//!
+//! [RFC-103](https://polkadot-fellows.github.io/RFCs/approved/0103-introduce-core-index-commitment.html) enables
+//! parachain runtimes to constrain the execution of each block to a specified core. This ensures
+//! better security and liveness properties as described in the RFC. To make use of this feature,
+//! the `SelectCore` trait needs to be implemented.
+#![doc = docify::embed!("../../cumulus/pallets/parachain-system/src/lib.rs", SelectCore)]
+//!
+//! For the vast majority of use cases, you will not need to implement a custom core
+//! selector. There are two pre-defined core selection policies to choose from
+//! `DefaultCoreSelector` and `LookaheadCoreSelector`.
+//!
+//! - The `DefaultCoreSelector` implements a round-robin selection on the cores that can be
+//! occupied by the parachain at the very next relay parent. This is the equivalent to what all
+//! parachains on production networks have been using so far.
+//!
+//! - The `LookaheadCoreSelector` also does a round robin on the assigned cores, but not those that
+//! can be occupied at the very next relay parent. Instead, it uses the ones after. In other words,
+//! the collator gets more time to build and advertise a collation for an assignment. This makes no
+//! difference in practice if the parachain is continuously scheduled on the cores. This policy is
+//! especially desirable for parachains that are sharing a core or that use on-demand coretime.
+//!
+//! In your /runtime/src/lib.rs, define a `SelectCore` type and use this to set the `SelectCore`
+//! property (overwrite it with the chosen policy type):
+#![doc = docify::embed!("../../templates/parachain/runtime/src/lib.rs", default_select_core)]
+//! ```ignore
+//! impl cumulus_pallet_parachain_system::Config for Runtime {
+//! ...
+//!     type SelectCore = SelectCore<Runtime>;
+//! ...
+//! }
+//! ```
+//!
+//! Next, we need to implement the `GetCoreSelector` runtime API. In the `impl_runtime_apis` block
+//! for your runtime, add the following code:
+//!
+//! ```ignore
+//! impl cumulus_primitives_core::GetCoreSelectorApi<Block> for Runtime {
+//! 		fn core_selector() -> (cumulus_primitives_core::CoreSelector, cumulus_primitives_core::ClaimQueueOffset) {
+//! 			ParachainSystem::core_selector()
+//! 		}
+//! }
+//! ```
+//!
+//! ### Phase 3 - Configure maximum scaling factor in the runtime
```

Comment:
can't we link some sample test parachain already configured and a zombienet test from CI suite ?

Reply:
added